README.md

```diff
@@ -579,6 +579,10 @@ This scene shows how to use ManualCameraControl.cs.  The script is on the main c
 
 This scene demonstrates how to use TextToSpeechManager.cs.  The script is placed on 3 cubes in the scene. Whenever a cube is activated with an air tap, a text to speech voice will emanate from the cube. The user can also ask "What time is it?" to hear the current time from a voice that stays with the user as they move.
 
+#### WindowOcclusion.unity 
```

Comment:
Please update the description for the Shader as well. Specially, the difference between Occlusion and WindowOcclusion.